<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsFeedPostView</class>
 <widget class="QMainWindow" name="NewsFeedPostView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QtWidgetListView" name="commentsListView">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="commentText"/>
      </item>
      <item>
       <widget class="QPushButton" name="commentButton">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Go_to_post"/>
    <addaction name="action_Synchronise"/>
    <addaction name="actionS_ettings"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Go_to_post">
   <property name="text">
    <string>&amp;Go to post</string>
   </property>
  </action>
  <action name="action_Synchronise">
   <property name="text">
    <string>&amp;Synchronise</string>
   </property>
  </action>
  <action name="actionS_ettings">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtWidgetListView</class>
   <extends>QListView</extends>
   <header>qtwidgetlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
